app/workers/import_worker.rb

```diff
@@ -0,0 +1,15 @@
+class ImportWorker
+  include Sidekiq::Worker
+
+  def perform(device, json_file)
```

Comment:
The underlying logic here needs to change anyway as far too slow for thousands of rows. But will refactor anyway as above for now.